File: Challenge 1/challenge1.cs
int CalculateEvenSum(int limit) {
    List<int> evenNumbers = new List<int>();

    for (int i = 1; i <= limit; i++) {
        if (i % 2 == 0) {
            evenNumbers.Add(i);
        }
    }

    int totalSum = 0;
    foreach (int num in evenNumbers) {
        totalSum += num;
    }

    return totalSum;
}

File: Challenge 1/challenge1.py
def calculate_even_sum(limit):
    even_numbers = []
    
    for i in range(1, limit + 1):
        if i % 2 == 0:
            even_numbers.append(i)
    
    total_sum = 0
    for num in even_numbers:
        total_sum += num
        
    return total_sum

print(calculate_even_sum(1000000))
